x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/aggregate-metrics/100_synthetic_source.yml

```diff
@@ -102,3 +112,12 @@ aggregate_metric_double with ignore_malformed:
             field: "field"
           value_count: 50
 
+  - do:
+      get:
+        index: test
+        id: "2"
+
+  - match:
+      _source:
+        metric: [{"min": 18.2,"max": 100.0, "value_count": 1}, "hey", 123, 456]
```

Comment:
Similar comment about missing the `[123, 456]` pair.

Reply:
See #109882.